### PR TITLE
new models

### PR DESCRIPTION
**Pull request by alanvgreen**
Adds new models and provides a fix to the gateware to allow those models to work correctly. Includes the changes to nextpnr-nexus params as suggested by Antmicro.

### CODE REVIEW

**Review by danc86 — APPROVED** [review]

File: scripts/parallel-nextpnr-nexus [hunk 1]
```diff
@@ -54,13 +55,14 @@ set -o pipefail
...
 mkdir runs
-seed_base=$((RANDOM % 100 * 100))
+seed_base=1000
 for s in $(seq ${SEED_COUNT}); do
   seed=$((seed_base + s))
   DIR="runs/seed-${seed}"
   mkdir "${DIR}"
   ( 
     nextpnr-nexus \
+        --placer-heap-timingweight 52 \
```
Comment (tcal-x) [inline scripts/parallel-nextpnr-nexus:65]: I think the new constant is 79, but we can do that in another PR.

**Review by tcal-x — APPROVED** [review]
LGTM
